localsettings/core.txt

```diff
@@ -0,0 +1,46 @@
+// Generic verbose error reporting stuff
+error_reporting( -1 );
+ini_set( 'display_startup_errors', 1 );
+ini_set( 'display_errors', 1 );
+$wgShowSQLErrors = true;
+$wgDebugDumpSql  = true;
+$wgShowDBErrorBacktrace = true;
+$wgShowExceptionDetails = true;
+$wgDebugComments = true;
+
+// Slice off the '/w' from the end
+$basePath = substr( $wgScriptPath, 0, -2 );
+
+// Don't share cookies across wikis (#78)
+$wgCookiePath = $basePath;
+
+// Short URLs
+$wgArticlePath = $basePath . "/wiki/$1";
+
+// Disable ResourceLoader's localStorage cache as this will
+// fill up quickly when domains host multiple wikis (#35)
+$wgResourceLoaderStorageEnabled = false;
+
+// Set the name of the project namespace to "Project:", rather than
+// "Patch Demo (123456,7):"
+$wgMetaNamespace = 'Project';
+
+// Special:Upload (also UploadWizard)
+$wgEnableUploads = true;
+
+// Logo
+$wgLogos = [
+	'svg' => "$wgResourceBasePath/logo.svg",
+	'wordmark' => [
+		'src' => "$wgResourceBasePath/wordmark.svg",
+		'width' => 190,
+		'height' => 50,
+	]
+];
+$wgFavicon = "$wgResourceBasePath/favicon.ico";
+
+// Email settings
+$wgAllowHTMLEmail = true;
+
+// Interwiki
```

Comment:
Should this be in an extension-specific file too?

Reply:
Yes.